benchmarks/bench_isolation_forest.py

```diff
@@ -0,0 +1,108 @@
+"""
+==========================================
+IsolationForest benchmark
+==========================================
+
+A test of IsolationForest on classical anomaly detection datasets.
+
+"""
+print(__doc__)
+
+import numpy as np
+from sklearn.ensemble import IsolationForest
+from sklearn.metrics import roc_curve, auc
+from sklearn.datasets import load_kddcup99, load_shuttle, fetch_covtype
+from sklearn.preprocessing import LabelBinarizer
+import matplotlib.pyplot as plt
+
+np.random.seed(1)
+
+
+# TODO: add to datasets 'shuttle', 'forestcover'
```

Comment:
Yes but the 4 you have in mind all come from the same dataset (kdd99). This TODO is actually out of date, it has already been done